### DISCUSSION TITLE
How can I get all week numbers for a given month?

### DISCUSSION THREAD

**Discussion by giri-jeedigunta** [jama5262/jiffy#162] [Q&A]
Hello !!!

I'm new to Jiffy. So far I'm loving it... I have a use case for my app where I need all the week numbers for a given month ... wondering what is the best way to do it with Jiffy... any inputs are highly appreciated. thanks !

**Comment by jama5262** [comment]
Hey @giri-jeedigunta, I'm having a hard time understanding your question about getting the week numbers for a given month using Jiffy. Could you give me an example?

Once I have a better understanding of what you're looking for, I'll be able to give you a more specific answer.

**Comment by jama5262** [comment]
@giri-jeedigunta This question has been dull for a long time now, please feel free to reopen this discussion later
